test/s3_test.go

```diff
@@ -122,8 +131,8 @@ func TestTerraformS3Module(t *testing.T) {
 			t.Parallel()
 
 			// These will create a tempTestFolder for each bucketTestCase.
-			tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "..", "test_examples/minimal")
-			roleTempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "..", "test_examples/helpers/iam_lpp")
+			tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "..", testCase.tfDir)
+			// roleTempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "..", "test_examples/helpers/iam_lpp")
```

Comment:
please just remove unused lines

Reply:
this was the only one, and it was removed on [cab9ea9](https://github.com/Datatamer/terraform-aws-s3/pull/15/commits/cab9ea98cae8259c6978d1ce0da95c2ef8ae48fb)